model/src/main/resources/reference.yml

```diff
@@ -31,6 +31,9 @@ ort:
   enableRepositoryPackageConfigurations: true
   enableRepositoryPackageCurations: true
 
+  # Force overwriting of any existing output files
```

Comment:
Please add a dot at the end of this sentence. Do this by amending the commit, but by adding a new commit that just adds the dot.

Reply:
> by adding a new commit that just adds the dot.

@dineshr93 you didn't amend you existing commit, but you added another commit. That is, there are not two commit on one in this PR. We'd like review comments to be addressed in the commit that the review comment belongs to.

Reply:
> by adding a new commit that just adds the dot.

@sschuberth I thought you were asking me to add a new commit that just adds the dot. but any way I have now made into single commit.

Reply:
Sorry, my original post was missing the "not". I've added it for reference.